src/overseer/api.clj

```diff
@@ -20,46 +20,38 @@
   start
   worker/start!)
 
-(def
-  ^{:doc "Construct a single unstarted job txn, given a type and optional
-  arguments, asserted as attributes on the job entities
+(defn job-graph
+  "Given a graph of job type keywords, and an optional map of
+  additional job data, return a graph of job maps that can be
+  transacted by a store.
+
+  This assumes you are only generating one job per type.
+
+  The input graph is specified in Loom adjacency list format (of keywords),
+  and the output is a Loom graph of job maps.
+
   Ex:
-    (let [tx1 (job-txn :my-job-type-1)
-          tx2 (job-txn :my-job-type-2 {:organization-id 123})]
-      @(d/transact conn [tx1 tx2]))"
-    :arglists '([job-type] [job-type args])}
-  job-txn
-  core/job-txn)
-
-(defn graph-txns
-  "Entry point to assert a sequence of jobs into the system.
-   Given a job graph, and optional additional argument data,
-   return a sequence of Datomic transactions that can be
-   asserted directly
-
-   Ex:
-     (def txns (graph-txns
-                 {:start []
-                  :process-1a [:start]
-                  :process-1b [:process-1a]
-                  :process-2 [:start]
-                  :finish [:process-1b :process-2]}
-                 {:organization-id 123}))
-     @(d/transact conn txns)"
+    (def graph (job-graph
+                {:start []
+                 :process-1a [:start]
+                 :process-1b [:process-1a]
+                 :process-2 [:start]
+                 :finish [:process-1b :process-2]}
+                {:org/id 123}))
+    ;; => Graph<Job> equivalent to:
```

Comment:
nit: I often read `equivalent to` in its weaker form as `functionally equivalent to`; I think `of the form` might be more exact/clear here